helm/default-apps-cloud-director/values.schema.json

```diff
@@ -1,142 +1,465 @@
 {
-    "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
@uvegla Sorry for the late feedback. Since we don't have proper tooling to manage this schema, we maintain it manually. Can you just add the new app as others manually? In this PR, you are removing the required section.

Reply:
Sure, checking!

Reply:
Updated. 👍🏻 Looks better, thanks for spotting this! 🙇🏻